synapseclient/retry.py

```diff
@@ -112,11 +112,8 @@ def _get_message(response):
     getting body.
     """
     if _is_json(response.headers.get('content-type', None)):
-        try:
-            json = response.json()
-            return json.get('reason', None)
-        except (AttributeError, ValueError):
-            pass
```

Comment:
The response can be truncated. So in this except we should return None. `pass` means that the function returns None here. But having a return statement will make the code more readable.